src/commands/PingCommand.ts

```diff
@@ -16,11 +17,22 @@ export class PingCommand implements ICommand {
     const greetings = [
       'Aye', "What's up",
       'Hi', 'Hello',
-      'Hiya', 'Hola'
+      'Hiya', 'Hola',
+      'Hi there', 'Fakka',
+      'Здравствуйте', '你好'
     ];
 
-    const { displayName } = message.guild.member(message.author);
```

Comment:
I recommend leaving this line in place. Otherwise, the bot refers to your `username` instead of your `displayName`.